docs/configuration.md

```diff
@@ -1,32 +1,50 @@
+|------------------------------|-----------|--------------------------------------------------------------------------------------------------|---------------------|------------------------------------------------------------------------|
+| Seed                         | int64     | any                                                                                              | 42                  | Seed for specification generator (for reproducibility)                 |
+| Platform                     | string    | Knative, Knative-RPS, OpenWhisk, OpenWhisk-RPS, AWSLambda, AWSLambda-RPS, Dirigent, Dirigent-RPS | Knative             | The serverless platform the functions will be executed on              |
+| YAMLSelector                 | string    | wimpy, container, firecracker                                                                    | container           | Service YAML depending on sandbox type                                 |
+| EndpointPort                 | int       | > 0                                                                                              | 80                  | Port to be appended to the service URL                                 |
+| RpsTarget                    | int       | > 0                                                                                              | 2                   | RPS-mode target load                                                   |
+| RpsColdStartRatioPercentage  | int       | >= 0 and <= 100                                                                                  | 50                  | RPS-mode cold/warm start ratio                                         |
+| RpsCooldownSeconds           | int       | > 0                                                                                              | 10                  | RPS-mode target load                                                   |
+| RpsImage                     | string    | "trace" or "empty"                                                                               | "empty"             | RPS-mode function image                                                |
+| RpsRuntimeMs                 | int       | > 0                                                                                              | 10                  | RPS-mode function runtime                                              |
+| RpsMemoryMB                  | int       | > 0                                                                                              | 2048                | RPS-mode function memory footprint                                     |
+| RpsIterationMultiplier       | int       | > 0                                                                                              | 80                  | RPS-mode SQRT iterations                                               |
```

Comment:
RPS mode is applicable to any system. As we don't read from the trace, we need to specify these 4 parameters. What alternative approach would you suggest?

Reply:
For knative deployment we have this kind stuff set in the yaml file. Maybe use similar approach for Dirigent? This would remove this parameter from the loader configuration and reduce confusion about the applicability of this parameter.

Another option is to make this parameter as a replacement for ITERATIONS_MULTIPLIER that we have in the yamls, this would be applicable to both modes and different platforms.